### DISCUSSION TITLE
请教搜索界面图1样式的设置项叫什么，图1用chrome+暴力猴，图2是edge+油

### DISCUSSION THREAD

**Discussion by bubbleboy11** [the1812/Bilibili-Evolved#3159] [提问 / Q&A] [answered]
![Snipaste_2022-03-22_20-14-53](https://user-images.githubusercontent.com/49860518/159480276-34352ab0-5570-493f-bcdd-d508b4ae8d2a.png)
![Snipaste_2022-03-22_20-14-29](https://user-images.githubusercontent.com/49860518/159480476-f93a87de-bcbc-4647-a6bd-1be00ced267e.png)

**Comment by the1812** [comment, marked as answer]
这是 b 站新版搜索界面, 你可以看右下角有个返回旧版

**Comment by bubbleboy11** [comment]
这个新版界面是随机用户拥有的权限吗，为什么一个账号有新版，另外一个用户没有

**Reply by the1812** [reply]
没错, b 站就喜欢玩这种, 叫灰度测试. 没被选中的人莫得选择

**Reply by bubbleboy11** [reply]
导出导入同一个evoled设置，居然界面不一样，我还对比设置一个个看看。。
